src/main/java/com/example/demo/manage/CrudUser.java

```diff
@@ -0,0 +1,62 @@
+package com.example.demo.manage;
+
+import java.util.List;
+//import java.util.Optional;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.example.demo.assembler.UserAssembler;
+import com.example.demo.model.User;
+import com.example.demo.response.dto.UserResponseDto;
+import com.example.demo.service.CrudService;
+
+
+//Create/Update/Delete User by ID
```

Comment:
- Write proper comments and remove unused code always.

- Always do CTRL + A,O,F before submitting code

Reply:
Removed unused Code

Reply:
do CTRL + A,O,F before submitting code and write proper comment on line no. 24